gtfs-realtime/CHANGES.md

```diff
@@ -117,4 +117,5 @@ message MyTripDescriptorExtension {
 |1008|SEPTA - Southeastern Pennsylvania Transportation Authority|[Gregory Apessos](mailto:GApessos@septa.org)|https://github.com/septadev|
 |1009|Swiftly|[mike@goswift.ly](mailto:mike@goswift.ly)|[Group Discussion](https://groups.google.com/forum/#!msg/gtfs-realtime/mmnZV6L-2ls/wVWdknhLBwAJ)|
 |1010|IBI Group|[Ritesh Warade](mailto:transitrealtime@ibigroup.com)|[GitHub proposal for new timestamps in Service Alerts](https://github.com/google/transit/pull/134)|
-|9000-9999|RESERVED - INTERNAL USE ONLY|[GTFS Community](https://groups.google.com/forum/#!forum/gtfs-realtime)|[Group discussion](https://github.com/google/transit/pull/178/)|
+|1013|MITFAHR⏐DE⏐ZENTRALE(MFDZ)|[Holger Bruch](mailto:holger.bruch@mfdz.de)|[Group discussion](https://groups.google.com/g/gtfs-realtime/c/IxYh-beoNoo)|
```

Comment:
I'm curious: Who uses `1011` and `1012`? (I don't care of course which ID MFDZ gets, I just wonder if there are existing stakeholders where it would be relevant to know what they're using the extensions for?)

Reply:
@derhuerst Currently, no one. MFDZ was assigned `1013` because of an error where we thought a stakeholder wanted 2 IDs for realtime extensions that were in fact GTFS schedule extensions. The next 2 extensions will have the ids `1011` and `1012`.